README.md

```diff
@@ -1853,6 +1865,8 @@ hz.getReliableTopic("my-distributed-topic").then(function (t) {
 });
 ```
 
+Hazelcast Reliable Topic uses `MessageListener` to listen to events that occur when a message is received. See the [Message Listener section](#7524-message-listener) for information on how to create a message listener object and register it.
```

Comment:
Hazelcast Reliable Topic uses `MessageListener` to listen to the events that occur when a message is received.